development/README.md

```diff
@@ -29,13 +29,13 @@ sudo systemctl reboot
 # Enabling services, but let's hold off on enabling redboot-auto-reboot.service for a bit
 # so we don't get into reboot-looping through all our boot attempts (i.e. until boot_counter reaches 0)
 sudo systemctl enable \
-    greenboot \
+    greenboot-task-runner \
     greenboot-healthcheck \
     greenboot-rpm-ostree-grub2-check-fallback \
     greenboot-grub2-set-counter \
     greenboot-grub2-set-success \
     greenboot-status \
-    redboot && \
+    redboot-task-runner && \
```

Comment:
We might want to enable `redboot-auto-reboot` as well, since that's the default in FIoT